src/components/Modal/Strategies.vue

```diff
@@ -18,7 +18,7 @@
           >
             <span v-text="key" class="flex-auto text-gray mr-1" />
             <a
-              v-if="key === 'address'"
+              v-if="key === 'address' || option.length === 42"
```

Comment:
How about using `isAddress` function instead here?

Reply:
Oh nice, added it